pkgs/by-name/md/mdbook-pdf-outline/package.nix

```diff
@@ -5,13 +5,13 @@
 }:
 
 python3Packages.buildPythonApplication rec {
-  pname = "mdbook-pdf-outline";
-  version = "0.1.4";
+  pname = "mdbook_pdf_outline";
```

Comment:
The change of package source is non-trivial (as `pname` here is also taken by `fetchPyPI` to form `src`), but there lacks explanation in the commit description and the PR description.

Could you elaborate the reason behind such change?

Reply:
@SuperSandro2000 Do you think we should change the `pname` styling of a Python package upon the upstream tarball name changes, or define the `pname` for `fetchPypi` separately?

Reply:
```suggestion
  pname = "mdbook-pdf-outline";
```

Reply:
Fixed